src/v/pandaproxy/test/pandaproxy_fixture.h

```diff
@@ -30,9 +30,9 @@ class pandaproxy_test_fixture : public redpanda_thread_fixture {
     pandaproxy_test_fixture operator=(pandaproxy_test_fixture&&) = delete;
     ~pandaproxy_test_fixture() = default;
 
-    http::client make_client() {
+    http::client make_client(uint16_t port = 8082) {
```

Comment:
we rarely use default args on functions, because they often mean different things. ok here, but why not have an explicit argument tho? probably just means changing in a couple of call sites no? 

just a nit.